python/ray/serve/_private/deployment_state.py

```diff
@@ -464,13 +464,17 @@ def start(self, deployment_info: DeploymentInfo) -> ReplicaSchedulingRequest:
             rank = 0
             init_args = (
                 self.replica_id,
-                cloudpickle.dumps(deployment_info.replica_config.deployment_def)
-                if self._deployment_is_cross_language
-                else deployment_info.replica_config.serialized_deployment_def,
+                (
```

Comment:
There seem to be some unrelated styling changes in this file; let's revert those for ease of review.

Reply:
reverted.